docs/examples/bifacial/plot_bifi_model_mc.py

```diff
@@ -0,0 +1,96 @@
+"""
+Bifacial Modeling - modelchain
```

Comment:
I think this example would benefit from having the code split into two smaller sections instead of one monolith of code.  The first code block would be what is interesting about this example (getting to the `effective_irradiance` variable) and the second would be more or less pvlib as usual (`array`, `system`, `mc_bifi`).